src/main/java/com/mmaarten/mmcommands/MmCommandHandler.java

```diff
@@ -40,273 +47,47 @@
-     * and the {@link MmCommandSignature#arguments() arguments}. The string will be used
-     * once in every help entry, not in the {@link #helpHeader(String) header}.
-     * <p>
-     * The last {@link ChatColor} code in this string will be the color
-     * used to display the {@link MmCommandSignature#arguments() arguments}.
-     * Default: "&amp;5"
-     * <p>
-     * ! Only used in the default implementation of {@link #printHelp(CommandSender, String, List, int)} !
-     *
-     * @param argumentSpacer the argument spacer
-     * @return this handler
-     */
-    public @NotNull MmCommandHandler helpArgumentSpacer(@NotNull String argumentSpacer) {
-        if (this.commands.size() != 0)
-            throw new RuntimeException("Options cannot be modified after commands have been added");
-        this.helpCommandArgumentSpacer = argumentSpacer;
-        return this;
-    }
-
-    /**
-     * Set the spacer to use in between the {@link MmCommandSignature#arguments() arguments}
-     * and the {@link MmCommandSignature#description() description}.
-     * The string will be used once in every help entry,
-     * not in the {@link #helpHeader(String) header}.
-     * <p>
-     * The last {@link ChatColor} code in this string will be the color
-     * used to display the {@link MmCommandSignature#description() description}.
-     * Default: "&amp;5"
-     * <p>
-     * ! Only used in the default implementation of {@link #printHelp(CommandSender, String, List, int)} !
-     *
-     * @param descriptionSpacer the description spacer
-     * @return this handler
-     */
-    public @NotNull MmCommandHandler helpDescriptionSpacer(@NotNull String descriptionSpacer) {
-        if (this.commands.size() != 0)
-            throw new RuntimeException("Options cannot be modified after commands have been added");
-        this.helpArgumentDescriptionSpacer = descriptionSpacer;
-        return this;
-    }
-
-    /**
-     * Set the prefix to use in the per command help menu
-     * The string will be used once in front of every property,
-     * not in the {@link #helpHeader(String) header}.
-     * <p>
-     * The last {@link ChatColor} code in this string will be the color
-     * used to display the property.
-     * Default: "&amp;d"
-     * <p>
-     * ! Only used in the default implementation of {@link #printPerCommandHelp(CommandSender, String, MmCommand)} !
-     *
-     * @param helpPropertyPrefix the property prefix
-     * @return this handler
-     */
-    public @NotNull MmCommandHandler helpPropertyPrefix(@NotNull String helpPropertyPrefix) {
-        if (this.commands.size() != 0)
-            throw new RuntimeException("Options cannot be modified after commands have been added");
+        this(Collections.<MmCommand>emptyList(), 5, false, false, false, "----Help" +
+                "---- " +
+                "page: " +
+            "%page%", "&d", " &5", " &8> &7", "&5", " &8> &7");
+    }
```

Comment:
Creating a default instance of the MmCommandHandler should be achieved by running build() on a non-modified builder directly. There should be no public constructor here. 
*side note*: try keeping string arguments on one line unless absolutely impossible.